src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -52,6 +52,10 @@
 
 namespace chip {
 
+namespace Crypto {
+class SessionKeystore;
```

Comment:
Why an anonymous pre-declaration? WHy can CHIPDeviceControllerSystemState not include a header that defnes SessionKeystore in full?

Reply:
It can, but using forward declarations where they are sufficient is a good practice as it reduces compilation time.